Source/JavaScriptCore/assembler/CPU.cpp

```diff
@@ -107,6 +107,20 @@ bool isARM64_LSE()
 }
 #endif
 
+#if CPU(ARM64E)
+bool isARM64E_FPAC()
+{
+#if OS(DARWIN)
+    uint32_t val = 0;
+    size_t valSize = sizeof(val);
+    int rc = sysctlbyname("hw.optional.arm.FEAT_FPAC", &val, &valSize, nullptr, 0);
+    return rc < 0 ? false : !!val;
+#else
+    return false;
+#endif
```

Comment:
Cache the result via `std::call_once` etc. here since `sysctlbyname` is costly (sys call).
These CPU's functions are designed to be called frequently from Air's lowering. So we need to make them cheap.

Reply:
Done.